### PR TITLE
Add multi-arch mng for test cluster

### PR DESCRIPTION
**Pull request by vikasmb**
-Also added verification steps for sed replacements

**What type of PR is this?**
feature

**Which issue does this PR fix**:
Adds verification for sed replacement and brings up 2 MNGs, one each for x86 and arm64 in integ test clusters.

**What does this PR do / Why do we need it**:
Adds verification for sed replacement and brings up 2 MNGs, one each for x86 and arm64 in integ test clusters.

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
[#1689](https://github.com/aws/amazon-vpc-cni-k8s/issues/1689)

**Testing done on this change**:

**Automation added to e2e**:

** Will this PR introduce any new dependencies?**:

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/run-integration-tests.sh [hunk 1]
```diff
@@ -185,11 +187,16 @@ echo "Using $BASE_CONFIG_PATH as a template"
 cp "$BASE_CONFIG_PATH" "$TEST_CONFIG_PATH"
 
 # Daemonset template
+# Replace image value and tag in cni manifest and grep (to verify that replacement was successful)
 echo "IMAGE NAME ${IMAGE_NAME} "
 sed -i'.bak' "s,602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni,$IMAGE_NAME," "$TEST_CONFIG_PATH"
+grep -r -q $IMAGE_NAME $TEST_CONFIG_PATH
+echo "Replacing manifest image tag $MANIFEST_IMAGE_VERSION with image version of $TEST_IMAGE_VERSION"
```
Comment (orsenthil) [inline scripts/run-integration-tests.sh:194]: Good logging. Thank you.

**Review by orsenthil — APPROVED** [review]
LGTM.  A result output give more confidence too (since this it is bash change).

**Comment by vikasmb** [comment]
> LGTM. A result output give more confidence too (since this it is bash change).

@orsenthil Successful test execution for this change is in https://github.com/aws/amazon-vpc-cni-k8s/runs/5043119996?check_suite_focus=true

**Review by jayanthvn — APPROVED** [review]
lgtm

**Pull request merged**